WeightProducer/python/MCSampleValues.py

```diff
@@ -251,669 +247,501 @@ class MCSampleValuesHelper():
             "CrossSection" : XSValues(
-                XS_2017=315.2, XSSource_2017="GenXSecAnalyzer",
-                XS_2018=314.6, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=5.085, XSSource_13TeV="XSDB (LO)",
             ),
         },
-        "WJetsToQQ_HT600to800_qc19_3j" : {
+        "TTGamma_Hadronic" : {
             "CrossSection" : XSValues(
-                XS_2017=68.61, XSSource_2017="GenXSecAnalyzer",
-                XS_2018=68.58, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=4.178, XSSource_13TeV="XSDB (LO)",
             ),
         },
-        "WJetsToQQ_HT-800toInf_qc19_3j" : {
+        "TTGamma_Dilept" : {
             "CrossSection" : XSValues(
-                XS_2017=34.71, XSSource_2017="GenXSecAnalyzer",
-                XS_2018=34.74, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=1.502, XSSource_13TeV="XSDB (LO)",
             ),
         },
-        "WJetsToQQ_HT-600ToInf" : {
-            "CrossSection" : XSValues(
-                XS_2016=99.65, XSSource_2016="XSDB (LO)"
+        "WJetsToLNu" : {
+           "CrossSection" : XSValues(
+                 XS_13TeV=53870.0, XSSource_13TeV="XSDB (LO)",
             ),
         },
-        "QCD_HT200to300" : {
+        "WJetsToLNu_HT-70To100" : {
             "CrossSection" : XSValues(
-                XS_2016=1710000.0, XSSource_2016="XSDB (LO)",
-                XS_2017=1547000.0, XSSource_2017="XSDB (LO)",
-                XS_2018=1557000.0, XSSource_2018="XSDB (LO)",
+                XS_13TeV=1264.0, XSSource_13TeV="XSDB (LO)",
             ),
+            #"kFactor" : kFactorValues(
+            #    kFactor_2016=1.224, kFactorSource_2016="XSDB NNLO/LO=61526.7/50260",
+            #    kFactor_2017=1.162, kFactorSource_2017="XSDB NNLO/LO=61526.7/52940",
+            #    kFactor_2018=1.164, kFactorSource_2018="XSDB NNLO/LO=61526.7/52850",
+            #),
         },
-        "QCD_HT300to500" : {
+        "WJetsToLNu_HT-100To200" : {
             "CrossSection" : XSValues(
-                XS_2016=347500.0, XSSource_2016="XSDB (LO)",
-                XS_2017=322600.0, XSSource_2017="XSDB (LO)",
-                XS_2018=323400.0, XSSource_2018="XSDB (LO)",
+                XS_13TeV=1256.0, XSSource_13TeV="XSDB (LO)",
             ),
+            #"kFactor" : kFactorValues(
+            #    kFactor_2016=1.224, kFactorSource_2016="XSDB NNLO/LO=61526.7/50260",
+            #    kFactor_2017=1.162, kFactorSource_2017="XSDB NNLO/LO=61526.7/52940",
+            #    kFactor_2018=1.164, kFactorSource_2018="XSDB NNLO/LO=61526.7/52850",
+            #),
         },
-        "QCD_HT500to700" : {
+        "WJetsToLNu_HT-200To400" : {
             "CrossSection" : XSValues(
-                XS_2016=32060.0, XSSource_2016="XSDB (LO)",
-                XS_2017=29980.0, XSSource_2017="XSDB (LO)",
-                XS_2018=30140.0, XSSource_2018="XSDB (LO)",
+                XS_13TeV=335.5, XSSource_13TeV="XSDB (LO)",
```

Comment:
Seems a lot lower than the EOY XS. Do you know why? Please double check this and the other WJetsToLNu_HT bins.

Reply:
I opened up a dialog in the GEN production hypernews [1]. Some reasons for the differences for WJets have been provided.

[1] https://hypernews.cern.ch/HyperNews/CMS/get/generators/5172.html

Reply:
Please collect these explanations and make a list of them (short summaries) in https://github.com/TreeMaker/TreeMaker/issues/584. That way I can refer to that issue when I'm next reviewing this PR and not have to go clicking through tons of links.